cmd/snap/cmd_validate.go

```diff
@@ -152,7 +151,15 @@ func (cmd *cmdValidate) Execute(args []string) error {
 			Mode:     action,
 			Sequence: seq,
 		}
-		return cmd.client.ApplyValidationSet(accountID, name, opts)
+		res, err := cmd.client.ApplyValidationSet(accountID, name, opts)
+		if err != nil {
+			return err
+		}
+		if res != nil && action == "monitor" {
+			fmt.Fprintln(Stdout, fmtValid(res))
+			return nil
```

Comment:
nitpick: this one could be removed
```suggestion
```

Reply:
I actually wanted to point out the same :-)